package.json

```diff
@@ -35,9 +33,6 @@
     "@octokit/request-error": "^2.1.0",
     "@octokit/rest": "^18.7.0",
     "@octokit/types": "^6.16.6",
-    "@types/browser-sync": "^2.26.3",
-    "@types/send": "^0.17.1",
-    "@types/tmp": "^0.2.1",
```

Comment:
I think all of these types are needed in the `http-server` which is built in consumer apps directly (so types are also needed 😄) We don't have a good strategy on Bazel-required NPM deps yet unfortunately.